src/utils/normalize-mime-parts.ts

```diff
@@ -64,7 +64,7 @@ export function getProfileImageUrl(
 ) {
 	return `${origin}/service/home/~/?max_width=100&max_height=100&auth=${
 		jwtToken ? 'jwt' : 'co'
-	}&id=${encodeURIComponent(profileImageId)}&t=${Date.now()}${
+	}&id=${encodeURIComponent(profileImageId)}${
```

Comment:
this means we have removed cache busting part, is this going to work even when images gets updated?

Reply:
Yes. For new image generate new/unique profileImageId.

Reply:
@silentsakky do you want to keep this change?

Reply:
does this gernerate new id for updated images?

Reply:
if we are having different profileImageId when updating image then it is fine to remove cache busting param